optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/AbstractUpgradeIT.java

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.upgrade;
```

Comment:
are there any changes in these test files that I should be aware of? Or are they all just uncommented?

Reply:
Just commenting them back in except for the template one where I also fixed the assertion.